packages/fake-backends/src/FakeChatThreadClient.ts

```diff
@@ -109,6 +109,10 @@ export class FakeChatThreadClient implements IChatThreadClient {
     const messages = this.checkedGetThread().messages;
     const message = messages[messages.length - 1];
 
+    if (!message) {
+      throw new Error(`${this.threadId} has no messages`);
+    }
+
```

Comment:
Not 100% sure but should we pass a empty array to the new steps instead?

Reply:
If there's no messages then something has gone wrong as this function has just emulated a sendMessage and stored that in the messages, so threw an error. open to changing though